cpu/efm32/include/periph_cpu.h

```diff
@@ -119,6 +120,18 @@ typedef struct {
 } dac_chan_conf_t;
 #endif
 
+/**
+ * @name    Real time counter configuration
+ * @{
+ */
+#define RTT_MAX_VALUE       _RTC_CNT_MASK        /* mask has all bits set ==> MAX*/
```

Comment:
`BOARD=sltb001a make -C tests/periph_rtt all ` does not run through for me:

```
"make" -C /home/chrysn/git/RIOT-doc-fixes/build/pkg/gecko_sdk/dist/emlib/src
In file included from /home/chrysn/git/RIOT-doc-fixes/boards/sltb001a/include/periph_conf.h:24,
                 from /home/chrysn/git/RIOT-doc-fixes/tests/periph_rtt/main.c:29:
/home/chrysn/git/RIOT-doc-fixes/tests/periph_rtt/main.c: In function 'cb':
/home/chrysn/git/RIOT-doc-fixes/cpu/efm32/include/periph_cpu.h:127:29: error: '_RTC_CNT_MASK' undeclared (first use in this function); did you mean 'RTCC_BASE'?
 #define RTT_MAX_VALUE       _RTC_CNT_MASK        /* mask has all bits set ==> MAX*/
                             ^~~~~~~~~~~~~
/home/chrysn/git/RIOT-doc-fixes/tests/periph_rtt/main.c:43:13: note: in expansion of macro 'RTT_MAX_VALUE'
     last &= RTT_MAX_VALUE;
```

Either something is wrong with the include sequence, or that mask is only defined for some of the gecko devices.

Reply:
seems like some call it _RTCC_CNT_MASK (maybe the same as the series 1/0 split)

Reply:
Seems like `_RTC_CNT_MASK ` is always 24 bit

Reply:
YOu could just do:

```
#ifdef _RTC_CNT_MASK 
#define RTT_MAX_VALUE       _RTC_CNT_MASK
#endif
#ifdef _RTCC_CNT_MASK 
#define RTT_MAX_VALUE       _RTCC_CNT_MASK 
#endif
```

If there is still another one the compiler will catch it.

Reply:
some BOARDS have 24Bit some 32Bit for RTT_MAX_VALUE (maybe the same as the RTC vs RTCC or series0 vs series1 cut)